test/extended/image_ecosystem/jenkins_plugin.go

```diff
@@ -225,42 +224,8 @@ var _ = g.Describe("[image_ecosystem][jenkins][Slow] openshift pipeline plugin",
 			o.Expect(err).NotTo(o.HaveOccurred())
 		}
 
-		jenkinsPod := jenkins.FindJenkinsPod(oc)
-
-		if os.Getenv(disableJenkinsMemoryStats) == "" {
-			ticker = time.NewTicker(10 * time.Second)
-			go func() {
-				for t := range ticker.C {
-					memstats, err := oc.Run("exec").Args("--namespace", jenkinsNamespace, jenkinsPod.Name, "--", "cat", "/sys/fs/cgroup/memory/memory.stat").Output()
-					if err != nil {
-						ginkgolog("\nUnable to acquire Jenkins cgroup memory.stat")
-					}
-					ps, err := oc.Run("exec").Args("--namespace", jenkinsNamespace, jenkinsPod.Name, "--", "ps", "faux").Output()
-					if err != nil {
-						ginkgolog("\nUnable to acquire Jenkins ps information")
-					}
-					ginkgolog("\nJenkins memory statistics at %v\n%s\n%s\n\n", t, ps, memstats)
-
-					// This is likely a temporary measure in place to extract diagnostic information during unexpectedly
-					// high memory utilization within the Jenkins image. If Jenkins is using
-					// a large amount of RSS, extract JVM information from the pod.
-					if memoryOverragePattern.MatchString(memstats) {
-						histogram, err := oc.Run("rsh").Args("--namespace", jenkinsNamespace, jenkinsPod.Name, "jmap", "-histo", "1").Output()
-						if err == nil {
-							ginkgolog("\n\nJenkins histogram:\n%s\n\n", histogram)
-						} else {
-							ginkgolog("Unable to acquire Jenkins histogram: %v", err)
-						}
-						stack, err := oc.Run("exec").Args("--namespace", jenkinsNamespace, jenkinsPod.Name, "--", "jstack", "1").Output()
-						if err == nil {
-							ginkgolog("\n\nJenkins thread dump:\n%s\n\n", stack)
-						} else {
-							ginkgolog("Unable to acquire Jenkins thread dump: %v", err)
-						}
-					}
-
-				}
-			}()
+		if os.Getenv(jenkins.DisableJenkinsMemoryStats) == "" {
+			ticker = jenkins.StartJenkinsMemoryTracking(oc, jenkinsNamespace)
```

Comment:
Yeah, it is a question of confidence that we aren't hitting the native memory issue any more.  If it proved to be very intermittent, and we didn't have this on when it happens again, that would be a bummer.

I'll defer to you and/or @jupierce on that.